### ISSUE TITLE
Use device instead of card in PCM()

### ISSUE THREAD

**Issue by amitay** [lintweaker/python-dsd-tools#4]
In pyalsaaudio >= 0.8.3, DSD formats are already added and the api for alsaaudio.PCM() has replaced the argument 'card' with 'device'.

Thanks for awesome work! 👍

**Comment by lintweaker** [comment]
Yes, thanks. They accepted my patches in 2015, it only took a while for a new (official) pyalsaaudio release. 
I had the 'device' for 'card' change locally for quite a while, never got around to update on git
